### PR TITLE
Students Id card Created.

### PR DESCRIPTION
**Pull request by BhagyashreeBhoyar**

### CODE REVIEW

File: src/components/SelectedStudentsActionWrapper.js [hunk 1]
```diff
@@ -1,8 +1,94 @@
 import React, {Component} from 'react';
+import Modal from 'react-modal';
 import { CSVLink } from 'react-csv';
+import { isEmpty } from 'simple-react-data-grid/src/utils/CommonUtils';
```
Comment (pepcus-pratik) [inline src/components/SelectedStudentsActionWrapper.js]: Use lodash's isEmpty method

File: src/components/SelectedStudentsActionWrapper.js [hunk 2]
```diff
@@ -1,8 +1,94 @@
...
+  };
+  openPrintOption() {
+    this.setState({printOptionIsOpen: true});
+  }
+  closePrintOption() {
+    this.setState({printOptionIsOpen: false});
+  }
+  renderStudentIdCards(student){
+   const studentsIdCards = student.map(object =>
+      <div className="student-Id-Cards">
```
Comment (pepcus-pratik) [inline src/components/SelectedStudentsActionWrapper.js]:
```suggestion
      <div className="student-id-cards">
```
Incorrect class name convention

File: src/components/SelectedStudentsActionWrapper.js [hunk 3]
```diff
@@ -1,8 +1,94 @@
...
+        <h2 className="student-Id-Cards-header"> Young Jain Study Group</h2>
+        <div className="card-fields-wrapper">
+          <div>Name: {object.name}</div>
+          <div>Class: {object.education}</div>
+          <div>Room: {object.classRoomNo2017}</div>
```
Comment (pepcus-pratik) [inline src/components/SelectedStudentsActionWrapper.js]: Add a TODO to use class room no of 2019

File: src/components/SelectedStudentsActionWrapper.js [hunk 4]
```diff
@@ -1,8 +1,94 @@
...
+          <div>Mobile No.: {object.mobile}</div>
+        </div>
+        <div className="card-fields-wrapper">
+          <div>Address: {object.address}</div>
+        </div>
+        <div className="card-fields-wrapper">
+          <div>Student </div>
+          <div>{object.studentId}</div>
+        </div>
+        <div className="student-Id-Cards-footer">
```
Comment (pepcus-pratik) [inline src/components/SelectedStudentsActionWrapper.js]: incorrect class name convention

File: src/components/SelectedStudentsActionWrapper.js [hunk 5]
```diff
@@ -1,8 +1,94 @@
...
+          <div>
+            Coordinator name: Bhagchand Jain
```
Comment (pepcus-pratik) [inline src/components/SelectedStudentsActionWrapper.js]: Add TODO to remove hard coded content

File: src/components/SelectedStudentsActionWrapper.js [hunk 6]
```diff
@@ -1,8 +1,94 @@
...
+          </div>
+          <div>
+            Coordinator contact: 8435534036
```
Comment (pepcus-pratik) [inline src/components/SelectedStudentsActionWrapper.js]: Add TODO to remove hard-coded content

File: src/components/SelectedStudentsActionWrapper.js [hunk 7]
```diff
@@ -1,8 +1,94 @@
...
+          </div>
+        </div>
+      </div>
+    );
+    if(!isEmpty(studentsIdCards)) {
+      return (
+        <div>
+          <h3>Students Id Cards</h3>
+          <div className="students-Id-Card-Wrapper">
```
Comment (pepcus-pratik) [inline src/components/SelectedStudentsActionWrapper.js]: incorrect class name convention

File: src/components/SelectedStudentsActionWrapper.js [hunk 8]
```diff
@@ -19,10 +105,23 @@ class SelectedStudentsActionWrapper extends Component{
           </CSVLink>
         </div>
         <div className="buttonContainer">
-          <button className="linkButton">
+          <button className="linkButton" onClick={this.openPrintOption}>
               <i className="fa fa-print card-icon"></i>Print Now
           </button>
         </div>
+        <Modal
```
Comment (pepcus-pratik) [inline src/components/SelectedStudentsActionWrapper.js]: Create a separate component for this modal

File: src/components/SelectedStudentsActionWrapper.js [hunk 9]
```diff
@@ -19,10 +105,23 @@ class SelectedStudentsActionWrapper extends Component{
...
+          isOpen={this.state.printOptionIsOpen}
+          onAfterOpen={this.afterOpenModal}
+          onRequestClose={this.closePrintOption}
+          style={customPrintOptionStyles}
+        >
+          <div > {this.renderStudentIdCards(this.props.selectedStudents)} </div>
```
Comment (pepcus-pratik) [inline src/components/SelectedStudentsActionWrapper.js]:
```suggestion
          <div>{this.renderStudentIdCards(this.props.selectedStudents)}</div>
```

Make sure avoid any extra white spaces in the code

File: src/components/SelectedStudentsActionWrapper.js (hunk 1 above)
Comment (BhagyashreeBhoyar) [reply src/components/SelectedStudentsActionWrapper.js]: Done.

File: src/components/SelectedStudentsActionWrapper.js (hunk 2 above)
Comment (BhagyashreeBhoyar) [reply src/components/SelectedStudentsActionWrapper.js]: Done.

File: src/components/SelectedStudentsActionWrapper.js (hunk 3 above)
Comment (BhagyashreeBhoyar) [reply src/components/SelectedStudentsActionWrapper.js]: Done.

File: src/components/SelectedStudentsActionWrapper.js (hunk 4 above)
Comment (BhagyashreeBhoyar) [reply src/components/SelectedStudentsActionWrapper.js]: Done.

File: src/components/SelectedStudentsActionWrapper.js (hunk 5 above)
Comment (BhagyashreeBhoyar) [reply src/components/SelectedStudentsActionWrapper.js]: Done.

File: src/components/SelectedStudentsActionWrapper.js (hunk 6 above)
Comment (BhagyashreeBhoyar) [reply src/components/SelectedStudentsActionWrapper.js]: Done.

File: src/components/SelectedStudentsActionWrapper.js (hunk 7 above)
Comment (BhagyashreeBhoyar) [reply src/components/SelectedStudentsActionWrapper.js]: Done.

**Pull request merged**